style.css

```diff
@@ -1,3 +1,82 @@
+html{
+    width: 375px;
+    height: 100%;
+    padding-left: 20px;
```

Comment:
- [ ] Also, make sure your website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/236639361-2f561415-9888-442d-8e40-9855994c4e51.PNG)| |![Capture](https://user-images.githubusercontent.com/57408419/236639285-50e747e4-bdd7-4c85-b298-a688523b9bb9.PNG)| |